src/views/Referrals/Referrals.tsx

```diff
@@ -0,0 +1,52 @@
+import { Content, Wrapper } from "./Referrals.styles";
+import {
+  RewardReferral,
+  RewardTableWithOverlay,
+  RewardMediumBlock,
+} from "./comp";
+import Footer from "components/Footer";
+import { useReferralsView } from "./useReferralsView";
+import { mediumUrl } from "utils";
+const Rewards = () => {
```

Comment:
nit: rename to fit the file name?
```suggestion
const Referrals = () => {
```